proxy/http/HttpTransactCache.cc

```diff
@@ -29,6 +29,10 @@
 #include <ctime>
 #include "HTTP.h"
 #include "HttpCompat.h"
+
+// inkcache
+#include "P_Cache.h"
```

Comment:
Can we get away with including `I_Cache.h` here?

Reply:
`I_Cache.h` doesn't bring in the symbols it needs.

Reply:
It was previously getting the symbols by a transient include of `P_Cache.h` in a header in `include/ts/InkAPIPrivateIOCore.h`.

Reply:
Otherwise I could include the specific headers it needs. `P_Cache.h` doesn't contain any declarations - it just brings in all the other Cache headers.

Reply:
We should stick to including the `I_` headers for includes outside of the library if we can.